docs/standard/native-interop/customizing-parameter-marshalling.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Customizing parameter marshalling
+description: Learn how to customize how .NET marshals your parameters to a native representation.
+author: jkoritzinsky
+ms.author: jekoritz
+ms.date: 11/28/2018
+---
+
+# Customizing parameter marshalling
+
+When the .NET runtime's default parameter marshalling behavior doesn't do what you want, use can use the <xref:System.Runtime.InteropServices.MarshalAsAttribute?displayProperty=nameWithType> attribute to customize how your parameters are marshaled.
```

Comment:
I am still seeing a lot of usage of the "you" pronoun. That should rarely be used in technical writing.

Reply:
> "do what you want, use can use"

"When the .NET runtime's default parameter marshalling behavior doesn't marshal in an expected manner, the ..."

Reply:
you is fine. we is the problem.